src/Components/Browser.JS/src/.eslintrc.js

```diff
@@ -0,0 +1,28 @@
+module.exports = {
+  parser: '@typescript-eslint/parser',  // Specifies the ESLint parser
+  plugins: ['@typescript-eslint'],
+  extends: [
+    'eslint:recommended',
+    'plugin:@typescript-eslint/recommended',  // Uses the recommended rules from the @typescript-eslint/eslint-plugin
+  ],
+  env: {
+    browser: true,
+    es6: true,
+  },
+  rules: {
+    // Place to specify ESLint rules. Can be used to overwrite rules specified from the extended configs
+    // e.g. "@typescript-eslint/explicit-function-return-type": "off",
+    "@typescript-eslint/indent": "off",
+    "no-var": "error",
+    "prefer-const": "error",
+    "quotes": ["error", "single", { "avoidEscape": true }],
+    "semi": "always",
+    "semi-style": ["error", "last"],
+    "semi-spacing": ["error", { "after": true }],
+    "spaced-comment": ["error", "always"],
+    "unicode-bom": "never"
+  },
+  globals: {
+    DotNet: "readonly"
+  }
+};
```

Comment:
Added ESLint to help with keeping the ts codebase clean.

Reply:
Great!